script/download-git.js

```diff
@@ -99,7 +99,15 @@ const downloadAndUnpack = () => {
   })
 }
 
-mkdirp(config.outputPath, function (error) {
+if (config.source === '') {
+  console.log(
+    `Skipping downloading embedded Git as platform '${process.platform}' is not supported.`
+  )
+  console.log(`To learn more about using dugite with a system Git: https://git.io/vF5oj`)
```

Comment:
This shortened URL will unwrap to `https://github.com/desktop/dugite/blob/master/docs/environment-variables.md#execution` which will work once the PR is merged.